spec/features/allocation_history_spec.rb

```diff
@@ -222,7 +222,15 @@
               expect(page).to have_css(key, text: val)
             end
           end
+        end
+      end
+
+      it 'has a Pentonville section', :js do
+        # 2nd Prison - Pentonville. This contains 7 events
```

Comment:
Now you have split the test into 2 does this test need this comment now? Is this now covered in the first test?